definitions/subworkflows/cellranger_mkfastq_and_count.cwl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "running cellranger mkfastq and count"
+
+inputs:
+    bcl_directory:
+        type: string
+    chemistry:
+        type: string
+    reference:
+        type: string
+    sample_name:
+        type: string
+    simple_sample_csv:
+        type: string
+    unique_id:
+        type: string
+
+steps:
+    mkfastq:
+        run: ../tools/cellranger_mkfastq.cwl
+        in:
+            bcl_directory: bcl_directory
+            simple_sample_csv: simple_sample_csv
+            unique_id: unique_id
+        out: [samplesheet_csv, fastq_dir]
+    count:
+        run: ../tools/cellranger_count.cwl
+        in: 
+            chemistry: chemistry
+            fastq_directory: mkfastq/fastq_dir
+            reference: reference
```

Comment:
Do we have a directory with the weird cellranger ref structure set up?  (I know I have one, but we should get one into /core/ for at least b38 and mm10.   Also, what's the position on including example YAMLs with new workflows.  I'm gonna go ahead and say it'd be a good idea to add one to this PR, if you have one!